cobra_test.go

```diff
@@ -42,9 +42,10 @@ var cmdEcho = &Command{
 }
 
 var cmdTimes = &Command{
-	Use:   "times [# times] [string to echo]",
-	Short: "Echo anything to the screen more times",
-	Long:  `a slightly useless command for testing.`,
+	Use:        "times [# times] [string to echo]",
+	Short:      "Echo anything to the screen more times",
+	Long:       `a slightly useless command for testing.`,
+	Deprecated: true,
```

Comment:
And use that in place of Use (or short or long)?  Should projects just change "Use" and we just not support 'deprecation' ?